python/rpdk/java/templates/init/guided_aws/StubUpdateHandler.java

```diff
@@ -0,0 +1,139 @@
+package {{ package_name }};
+
+// TODO: replace all usage of SdkClient with your service client type, e.g; YourServiceAsyncClient
+// import software.amazon.awssdk.services.yourservice.YourServiceAsyncClient;
+
+import software.amazon.awssdk.awscore.AwsRequest;
+import software.amazon.awssdk.awscore.AwsResponse;
+import software.amazon.awssdk.awscore.exception.AwsServiceException;
+import software.amazon.awssdk.core.SdkClient;
+import software.amazon.cloudformation.exceptions.CfnGeneralServiceException;
+import software.amazon.cloudformation.proxy.AmazonWebServicesClientProxy;
+import software.amazon.cloudformation.proxy.Logger;
+import software.amazon.cloudformation.proxy.ProgressEvent;
+import software.amazon.cloudformation.proxy.ProxyClient;
+import software.amazon.cloudformation.proxy.ResourceHandlerRequest;
+
+public class UpdateHandler extends BaseHandlerStd {
+    private Logger logger;
+
+    protected ProgressEvent<ResourceModel, CallbackContext> handleRequest(
+        final AmazonWebServicesClientProxy proxy,
+        final ResourceHandlerRequest<ResourceModel> request,
+        final CallbackContext callbackContext,
+        final ProxyClient<SdkClient> proxyClient,
+        final Logger logger) {
+
+        this.logger = logger;
+
+        final ResourceModel model = request.getDesiredResourceState();
+
+        // TODO: Adjust Progress Chain according to your implementation
+
+        // STEP 1 [initialize a proxy context]
+        return proxy.initiate("{{ call_graph }}::{{ operation }}", proxyClient, model, callbackContext)
+
+            // STEP 2 [TODO: construct a body of a request]
+            .translate(Translator::translateToUpdateRequest)
+
+            // STEP 3 [TODO: make an api call]
+            .call(this::updateResource)
+
+            // STEP 4 [TODO: stabilize step is not necessarily required but typically involves describing the resource until it is in a certain status, though it can take many forms]
+            // stabilization step may or may not be needed after each API call
+            // for more information -> https://docs.aws.amazon.com/cloudformation-cli/latest/userguide/resource-type-test-contract.html#resource-type-test-contract-communication
```

Comment:
again, remove the `#resource-type-test-contract-communication` from all the URLs because this will change